wire-grpc-tests/src/test/java/com/squareup/wire/GrpcClientTest.kt

```diff
@@ -576,6 +585,127 @@ class GrpcClientTest {
     }
   }
 
+  @Test
+  fun grpcCallIsCancelledWhenItShouldBe() {
```

Comment:
Nit: canceled with one L

Reply:
> Both spellings are correct; Americans favor canceled (one L), while cancelled (two Ls) is preferred in British English and other dialects. However, there is only one correct spelling of the word cancellation, no matter where you are.

Reply:
I wish canceled took 2 Ls and it's was the possessive of it.